### PR TITLE
 charts/{brig,galley}: Allow setting a preStop hook for the deployments

### PR DESCRIPTION
**Pull request by akshaymankar**
https://wearezeta.atlassian.net/browse/WPB-9075

## Checklist

 - [x] Add a new entry in an appropriate subdirectory of `changelog.d`
 - [x] Read and follow the [PR guidelines](https://docs.wire.com/developer/developer/pr-guidelines.html)

### CODE REVIEW

**Comment by akshaymankar** [comment]
The idea is we'll deploy profiling builds and set this value to upload the profile data somewhere. This was when k8s decides to kill the container, the profile data won't get lost.

File: charts/brig/values.yaml [hunk 1]
```diff
@@ -14,6 +14,9 @@ resources:
 metrics:
   serviceMonitor:
     enabled: false
+# This is not supported for production use, only here for testing:
+# preStop:
+#  exec: ["sh", "-c", "curl http://acme.example"]
```
Comment (lwille) [inline charts/brig/values.yaml:19]: **issue:** the structure suggested here is incorrect and does not validate.

```suggestion
#  exec: 
#   command: ["sh", "-c", "curl http://acme.example"]
```

File: charts/galley/values.yaml [hunk 2]
```diff
@@ -15,6 +15,9 @@ resources:
     cpu: "100m"
   limits:
     memory: "500Mi"
+# This is not supported for production use, only here for testing:
+# preStop:
+#  exec: ["sh", "-c", "curl http://acme.example"]
```
Comment (lwille) [inline charts/galley/values.yaml:20]: **issue:** the structure suggested here is incorrect and does not validate.

```suggestion
#  exec: 
#   command: ["sh", "-c", "curl http://acme.example"]
```

**Review by lwille — CHANGES_REQUESTED** [review]
just found two tiny issues

**Comment by lwille** [comment]
feel free to merge once CI is green :)

**Pull request merged**